test/plug/upload_test.exs

```diff
@@ -1,6 +1,30 @@
 defmodule Plug.UploadTest do
   use ExUnit.Case, async: true
 
+  defp spawn_lots_of_temporary_files(_stop_time, 0), do: :ok
+  defp spawn_lots_of_temporary_files(stop_time, reps) do
+    if :os.system_time(:milli_seconds) < stop_time do
+      {pid, ref} = spawn_monitor fn ->
+        {:ok, _} = Plug.Upload.random_file("sample")
+      end
+      spawn_lots_of_temporary_files(stop_time, reps - 1)
+
+      :ok =
+      receive do
+        {:DOWN, ^ref, :process, ^pid, :normal} -> :ok
+      after
+	20_000 -> :timed_out
```

Comment:
sorry, I usually use Linux, and my emacs is set up properly there. I made this on windows